django_mongoengine_filter/filterset.py

```diff
@@ -193,12 +194,13 @@ class BaseFilterSet(object):
     order_by_field = ORDER_BY_FIELD
     strict = True
 
-    def __init__(self, data=None, queryset=None, prefix=None, strict=None):
+    def __init__(self, data=None, queryset=None, *, request=None, prefix=None, strict=None):
```

Comment:
This mirrors the Filterset.__init__() function arguments that are in the django_filter.filterset.BaseFilterSet.__init__()